vscode-trace-extension/src/extension.ts

```diff
@@ -231,7 +250,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<Extern
     // Workaround for Theia compatibility of welcome screen. See https://github.com/eclipse-theia/theia/issues/9361
     context.subscriptions.push(vscode.window.registerTreeDataProvider('welcome', new EmptyTreeDataProvider()));
 
-    return traceExtensionAPI;
+    // Append Diagnostic to traceExtensionAPI
+    const diagnostics = messenger.diagnosticApi({ withParameterData: true, withResponseData: true });
+    return { ...traceExtensionAPI, ...diagnostics };
```

Comment:
I think we can skip returning the "diagnostics" if we are not using the "vscode-messenger devtools" library along with "vscode-messenger". Rollback these 3 lines

Reply:
I misunderstood- closing this comment.